Mailbox.Tests/MailBoxTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace Mailbox.Tests
+{
+    [TestClass()]
+    public class MailBoxTests
+    {
+        [TestMethod]
+        public void MailBoxConstrucotrTest_SizePremium()
+        {
+            //Arrange
+            Sizes size = Sizes.Premium;
+            ValueTuple<int, int> location = new ValueTuple<int, int>(15, 5);
```

Comment:
Tuple syntax can be simplified:
```suggestion
            (int, int) location = (15, 5);
```